src/sensor_state_data/library.py

```diff
@@ -144,6 +144,10 @@ class SensorLibrary:
         device_class=SensorDeviceClass.ROTATION,
         native_unit_of_measurement=Units.DEGREE,
     )
+    ROTATIONAL_SPEED__REVOLUTIONS_PER_MINUTE = BaseSensorDescription(
+        device_class=SensorDeviceClass.ROTATIONAL_SPEED,
```

Comment:
This device class doesn't appear to exist in Home Assistant

Reply:
I think we could accept the units here no problem.  The SensorLibrary shouldn't get a device class that doesn't exist though.